### ISSUE TITLE
Error during build

### ISSUE THREAD

**Issue by LuukHenk** [ratschlab/spladder#78]
* spladder version: 2.0.0
* Python version: 3.7
* Operating System: Ubuntu Linux 18.04

### Description

Running SplAdder build resulted in a NameError, because a variable in the init.py script is not defined. 

### What I Did

```
spladder build -o home/lperdaems/outputdir -a home/lperdaems/gencode.v27.annotation.gtf -b home/lperdaems/bamFiles/patient1.bam,home/lperdaems/bamFiles/patient2.bam,home/lperdaems/bamFiles/patient3.bam --parallel 3

Traceback (most recent call last):
  File "/home/lperdaems/anaconda3/bin/spladder", line 11, in <module>
    load_entry_point('spladder==2.0.0', 'console_scripts', 'spladder')()
  File "/home/lperdaems/anaconda3/lib/python3.7/site-packages/spladder/spladder.py", line 182, in main
    options.func(options)
  File "/home/lperdaems/anaconda3/lib/python3.7/site-packages/spladder/spladder_build.py", line 72, in spladder
    (genes, options) = init.init_genes_gtf(options)
  File "/home/lperdaems/anaconda3/lib/python3.7/site-packages/spladder/init.py", line 80, in init_genes_gtf
    for line in open(infile, 'r'):
NameError: name 'infile' is not defined
```

**Comment by akahles** [comment]
Dear @LuukHenk ,

Thank you very much for reporting. I have pushed a fix to the development branch. Could you please test whether this resolves your issue? If yes, I can push out a bugfix release.

Cheers,
Andre

**Comment by LuukHenk** [comment]
Dear Andre Kahles,

The bugfix resolved the issue, thanks!

Best regards,
Luuk Perdaems

________________________________
Van: Andre Kahles <notifications@github.com>
Verzonden: dinsdag 19 maart 2019 17:05
Aan: ratschlab/spladder
CC: Luuk Perdaems; Mention
Onderwerp: Re: [ratschlab/spladder] Error during build (#78)


Dear @LuukHenk<https://github.com/LuukHenk> ,

Thank you very much for reporting. I have pushed a fix to the development branch. Could you please test whether this resolves your issue? If yes, I can push out a bugfix release.

Cheers,
Andre

**Comment by akahles** [comment]
Thanks for confirming. Have released version 2.0.1.
